### PR TITLE
chore: rename folder testdata/gcb-with-location

### PR DESCRIPTION
**Pull request by ericzzzzzzz**
rename typo from #7411, not sure why the integration test passed though with incorrect path.

### CODE REVIEW

**Comment by ericzzzzzzz** [comment]
> > rename typo from #7411, not sure why the integration test passed though with incorrect path.
> 
> I think its because this test is GCP only and get executed in kokoro only. Please check the logs of kokoro job. image

seems kokoro failed before tests are running.

**Comment by tejal29** [comment]
> > > rename typo from #7411, not sure why the integration test passed though with incorrect path.
> > 
> > 
> > I think its because this test is GCP only and get executed in kokoro only. Please check the logs of kokoro job. image
> 
> seems kokoro failed before tests are running.

Can you manually run the test and paste the logs here? I can manually verify.

**Review by gsquared94 — APPROVED** [review]

**Comment by ericzzzzzzz** [comment]
Hi @tejal29  I manually change needsGCP to CanRunWithoutGCP, then run the test, test passes. 
```bash
/Users/ericwork/sdk/go1.17.9/bin/go tool test2json -t /private/var/folders/dd/3jk5b0ys5tldsws9kr06zxkr00yngh/T/GoLand/___TestRunGCPOnly_Google_Cloud_Build_with_location_in_github_com_GoogleContainerTools_skaffold_integration.test -test.v -test.paniconexit0 -test.run ^\QTestRunGCPOnly\E$/^\QGoogle_Cloud_Build_with_location\E$
=== RUN   TestRunGCPOnly
--- PASS: TestRunGCPOnly (57.69s)
=== RUN   TestRunGCPOnly/Google_Cloud_Build_with_location
time="2022-05-19T11:38:40-07:00" level=info msg="Namespace: skaffoldg6468" subtask=-1 task=DevLoop
    helper.go:259: Running [skaffold run --namespace skaffoldg6468 --default-repo gcr.io/k8s-skaffold] in testdata/gcb-with-location
    helper.go:259: Ran [skaffold run --namespace skaffoldg6468 --default-repo gcr.io/k8s-skaffold] in 54.231 seconds
time="2022-05-19T11:39:35-07:00" level=info msg="Waiting for pods [getting-started] to be ready" subtask=-1 task=DevLoop
time="2022-05-19T11:39:35-07:00" level=info msg="Pods marked as ready: map[getting-started:true]" subtask=-1 task=DevLoop
    helper.go:259: Running [skaffold delete --namespace skaffoldg6468 --default-repo gcr.io/k8s-skaffold] in testdata/gcb-with-location
    helper.go:259: Ran [skaffold delete --namespace skaffoldg6468 --default-repo gcr.io/k8s-skaffold] in 2.288 seconds
    --- PASS: TestRunGCPOnly/Google_Cloud_Build_with_location (57.69s)
PASS
```
here is the build history log
 https://pantheon.corp.google.com/cloud-build/builds;region=asia-east1/0bf54e0d-55b7-4a92-aff8-e733bb69a9ff?project=k8s-skaffold
 
 built image : https://pantheon.corp.google.com/gcr/images/k8s-skaffold/global/skaffold-example-with-location?project=k8s-skaffold

File: integration/testdata/gcb-with-location/skaffold.yaml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 apiVersion: skaffold/v3alpha1
 kind: Config
 build:
+  artifacts:
+    - image: gcr.io/k8s-skaffold/skaffold-example-with-location
   googleCloudBuild:
     projectId: k8s-skaffold
     region: asia-east1
```
Comment (tejal29) [inline integration/testdata/gcb-with-location/skaffold.yaml:8]: Verified region in the build job!

**Review by tejal29 — COMMENTED** [review]
Approved!!

**Pull request merged**